script.service.next-episode/libs/medialibrary.py

```diff
@@ -0,0 +1,172 @@
+        ``{u'season': 4, u'playcount': 0, u'episode': 1, u'episodeid': 5, u'label': u'4x01. The Drone Queen'}``
+    :rtype: list
+    :raises: NoDataError if a TV show has no episodes.
+    """
+    params = {'tvshowid': tvshowid, 'properties': ['season', 'episode', 'playcount', 'tvshowid']}
+    result = send_json_rpc('VideoLibrary.GetEpisodes', params)
+    if not result.get('episodes'):
+        raise NoDataError
+    return result['episodes']
+
+
+def get_tvdb_id(tvshowid):
+    """
+    Get TheTVDB ID for a TV show
+
+    :param tvshowid: internal Kodi database ID for a TV show
+    :type tvshowid: str
+    :return: TheTVDB ID
+    :rtype: str
+    """
+    params = {'tvshowid': tvshowid, 'properties': ['imdbnumber']}
+    return send_json_rpc('VideoLibrary.GetTVShowDetails', params)['tvshowdetails']['imdbnumber']
+
+
+def get_recent_movies():
+    """
+    Get the list of recently added movies
+
+    :return: the list of recent movies
+    :rtype: list
+    :raises: NoDataError if the Kodi library has no recent movies.
+    """
+    params = {'properties': ['imdbnumber', 'playcount']}
+    result = send_json_rpc('VideoLibrary.GetRecentlyAddedMovies', params)
+    if not result.get('movies'):
+        raise NoDataError
+    return result['movies']
+
+
+def get_recent_episodes():
+    """
+    Get the list of recently added episodes
+
+    :return: the list of recent episodes
+    :rtype: list
+    :raises: NoDataError if the Kodi library has no recent episodes
+    """
+    params = {'properties': ['playcount', 'tvshowid', 'season', 'episode']}
+    result = send_json_rpc('VideoLibrary.GetRecentlyAddedEpisodes', params)
+    if not result.get('episodes'):
+        raise NoDataError
+    return result['episodes']
+
+
+def get_now_played():
+    """
+    Get nov played item
+
+    :return: now played item's data
+    :rtype: dict
+
+    Example movie::
+
+        {u'tvshowid': -1, u'episode': -1, u'season': -1,
+            u'label': u'12 Years a Slave', u'imdbnumber': u'tt2024544',
+            u'playcount': 0, u'type': u'movie', u'id': 1}
+
+    Example episode::
+
+        {u'tvshowid': 1, u'episode': 11, u'season': 4,
+            u'label': u'The Path of Destruction', u'imdbnumber': u'',
+            u'playcount': 0, u'type': u'episode', u'id': 1}
+    """
+    playerid = -1
+    for player in send_json_rpc('Player.GetActivePlayers'):
+        if player['type'] == 'video':
+            playerid = player['playerid']
+            break
+    params = {'playerid': playerid, 'properties': ['playcount', 'imdbnumber', 'season', 'episode', 'tvshowid']}
+    return send_json_rpc('Player.GetItem', params)['item']
+
+
+def get_playcount(id_, type):
+    """
+    Get video item playcount
+
+    :param id_: movie or episode Kodi database ID
+    :type id_: int
+    :param type: items's type -- 'movie' or 'episode'
+    :type type: str
+    :return: playcount
+    :rtype: int
+    """
+    if type == 'movie':
```

Comment:
Actually, using a single trailing underscore to avoid name conflicts is officially recommended in PEP-8.